modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java

```diff
@@ -556,9 +572,17 @@ else if (Validator.isNotNull(stickerLabel)) {
 		jspWriter.write("<div class=\"card-body\"><div class=\"card-row\">");
 		jspWriter.write("<div class=\"autofit-col autofit-col-expand\">");
 
+		String ariaLabel = getAriaLabel();
+
+		jspWriter.write("<p class=\"card-title\" aria-label=\"");
+
```

Comment:
do we want to output empty aria-label here in case it's null? Probably a nit though

Reply:
I updated this case so it only adds the aria-label if the value is not null 😄